projects/plugins/jetpack/extensions/blocks/subscriber-login/subscriber-login.php

```diff
@@ -82,6 +82,15 @@ function get_subscriber_login_url() {
 	);
 }
 
+/**
+ * Determines whether the current visitor is a logged in user or a subscriber.
+ *
+ * @return bool
+ */
+function is_subscriber_logged_in() {
+	return is_user_logged_in() || Abstract_Token_Subscription_Service::has_token_from_cookie();
```

Comment:
Thinking if there's any edge case where user is logged in, but they're some bespoke or low-level user role which doesn't get access to anything really. 🤔 

We can iterate on that if it's an issue, and go with this for now. Solution could be to check if they have `manage_posts` capability.

Reply:
I think you're right. We show the user as logged in even though the email is not confirmed and the wp.com account is not created. Will add it to TODOs, thanks!